manga-loader.user.js

```diff
@@ -1150,6 +1150,32 @@ var implementations = [{
   curpage: 'a.onpage',
   numpages: function() {
     return extractInfo('select[name=jump]') - 1;
+  },
+  nextchap: function() {
+      return W._nextchap;
+  },
+  prevchap: function() {
+      return W._prevchap;
+  },
+  wait: function() {
+    if (!W._ajaxdone) {
+      W._ajaxdone = -1;
+      ajax({
+        method: 'GET',
+        async: false,
+        url: location.href.split('/')[4].match(/^[\d]{4}/)[0] + '.html',
+        onload: function(e) {
+          var res = e.target.response;
+          var chapters = res.match('<td>.(<a.*)</td>')[1].split('<td>').map(i => i.match(/href=(.*?) /)[1]);
```

Comment:
I'd prefer to keep the script es5 compatible for now (as much as I like es6+ syntax) . Mostly because the rest of the code is.